test/jdk/sun/security/ssl/SSLSocketImpl/SSLSocketLeak.java

```diff
@@ -53,7 +58,7 @@ public static void main(String[] args) throws IOException {
         long fds_end = FileUtils.getProcessHandleCount();
         System.out.println("FDs in the end: " + fds_end);
 
-        if ((fds_end - fds_start) > (NUM_TEST_SOCK / 10)) {
+        if ((fds_end - fds_start) > (NUM_TEST_SOCK / OPEN_HANDLE_GROWTH_THRESHOLD)) {
```

Comment:
It looks like the test may still fail intermittent.  But let's do it for now and see if it is reliable to be a automation test case.

Reply:
The failing cases I've seen in SAP's infrastructure have all been on Windows and the number of handles was marginally above the 10 percent threshold. Do you have seen otherwise in Oracle's tests? The test prints out the handle count before and after...

Reply:
The number may help for general cases, but intermittent failure could happen if the test could fail in theory.  I did not dig into Oracle testing result.

Reply:
As per Tony's [JBS comment](https://bugs.openjdk.java.net/browse/JDK-8257670?focusedCommentId=14387025&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14387025), the test issue was observed on Windows only also in Oracle and it indicates that the threshold of 10% is just missed by few - it would be 50 fds. So I'd hope we're good with this modification.